vars/withPipeline.groovy

```diff
@@ -194,6 +210,14 @@ def call(type, String product, String component, Closure body) {
             tfPlanOnly: false
           )
 
+          highLevelDataSetup(
```

Comment:
this is missing in the `onAutoDeployBranch` which means it won't work on branches like demo / ithc / perftest

Reply:
resolved in [b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/b7ebe7a5e40a9e22fc3300f30a00582f0183cfb1)